pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -561,5 +580,17 @@ func (c *Cluster) resolveNode(nodeName string) *rookalpha.Node {
 		return nil
 	}
 	rookNode.Resources = k8sutil.MergeResourceRequirements(rookNode.Resources, c.resources)
+
+	// ensure no invalid dirs are specified
```

Comment:
Is this mentioned in the upgrade guide?
If not, adding steps to change/move the data dir would be good.

Reply:
i believe these should already have been invalid in 0.7, i'll try it out